python/pyspark/errors/error_classes.py

```diff
@@ -29,6 +34,21 @@
       "Attribute `<attr_name>` in provided object `<obj_name>` is not callable."
     ]
   },
+  "BARRIER_TASK_CONTEXT_NOT_INITIALIZE": {
+    "message": [
+      "Not supported to call `<func_name>` before initialize BarrierTaskContext."
+    ]
+  },
+  "BROADCAST_VARIABLE_NOT_LOADED": {
+    "message": [
+      "Broadcast variable `<variable>` not loaded."
+    ]
+  },
+  "CALL_BEFORE_SESSION_INITIALIZE" : {
```

Comment:
what about combine `BARRIER_TASK_CONTEXT_NOT_INITIALIZE` and `CALL_BEFORE_SESSION_INITIALIZE` to a general `CALL_BEFORE_INITIALIZE` with an extra parameter `object`?